code/modules/mining/machine_vending.dm

```diff
@@ -129,7 +129,7 @@
 				return
 			to_chat(usr, "<span class='notice'>[src] clanks to life briefly before vending [prize.equipment_name]!</span>")
 			var/obj/created = new prize.equipment_path(loc)
-			if (M.CanReach(src))
+			if (M.CanReach(src) && istype(created, /obj/item))
```

Comment:
```suggestion
			if (M.CanReach(src) && isitem(created))
```

Reply:
DId not know that existed, thanks mr Caesar Salad